igvc_utils/include/igvc_utils/robot_control.h

```diff
@@ -19,7 +19,10 @@ class RobotControl
   friend std::ostream &operator<<(std::ostream &out, const RobotControl &robot_control);
 
   igvc_msgs::velocity_pair toMessage(ros::Time stamp) const;
-  CurvatureVelocity toKV(double axle_length = -1) const;
+  CurvatureVelocity toKV(const std::optional<double>& axle_length = std::nullopt) const;
+
+  double linearVelocity() const;
+  double angularVelocity(const std::optional<double>& axle_length = std::nullopt) const;
```

Comment:
Why would you ever need to call this with a null `axle_length`?

Reply:
I guess the thing I'm not sure about really is where `axle_length` should belong. It doesn't feel right that `axle_length` is a member of `RobotControl`, yet if it isn't then the caller needs to supply `axle_length` as an argument every time they want to convert from left / right velocities to curvature and linear velocity.

I guess the right thing to do here would be to just have the caller pass it every time, even though it's a pain to do.